logger/include/logger.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <filesystem>
+#include <iostream>
+#include <list>
+#include <map>
+#include <source_location>
+#include <sstream>
+
+enum Loglevel
+{
+    INFO,
+    CRITICAL
+};
+
+namespace Logger
+{
+
+// size of buffer is 8k
+const size_t bufferSize = 8000;
+inline std::list<std::string> buffer;
+static std::map<Loglevel, std::string> log = {{Loglevel::INFO, "INFO"},
+                                              {Loglevel::CRITICAL, "CRITICAL"}};
+/**
+ *@brief Appends data to the buffer
+ *
+ *This API is called to log message to the buffer.
+ *It checks the log level and appends information like
+ *line number, file name, timestamp to the buffer.
```

Comment:
All these data are getting appended to the message, not to the buffer. Please re-phrase.